src/components/Accordion/index.tsx

```diff
@@ -0,0 +1,34 @@
+import type {ReactNode} from 'react';
+import React from 'react';
+import {View} from 'react-native';
+import type {SharedValue} from 'react-native-reanimated';
+import Animated, {useAnimatedStyle, useDerivedValue, useSharedValue, withTiming} from 'react-native-reanimated';
+import useThemeStyles from '@hooks/useThemeStyles';
+
+function Accordion({isExpanded, children, duration = 300}: {isExpanded: SharedValue<boolean>; children: ReactNode; duration?: number}) {
```

Comment:
Let's extract props type to a separate type above, a good rule of thumb is to always do it for complex types like this